tools/tuf/download-artifacts/download-artifacts.go

```diff
@@ -138,9 +139,10 @@ func desktopCommand() *cli.Command {
 		},
 		Action: func(c *cli.Context) error {
 			return downloadComponents("generate-desktop-targets.yml", gitBranch, map[string]string{
-				"macos":   "desktop.app.tar.gz",
-				"linux":   "desktop.tar.gz",
-				"windows": "fleet-desktop.exe",
+				"macos":       "desktop.app.tar.gz",
+				"linux":       "desktop.tar.gz",
+				"linux-arm64": "desktop-arm64.tar.gz",
```

Comment:
Name of this file has to match what's in `releaser.sh` (which I believe it's just `desktop.tar.gz`), right?

Reply:
Am happy to merge this as is and fix/test this when releasing.